README.rst

```diff
@@ -266,10 +266,25 @@ of the endpoint parameter ``message`` to your view function.
 
 .. note:: In the OpenAPI 3.x.x spec, the requestBody does not have a name.
           By default it will be passed in as 'body'. You can optionally
-          provide the x-body-name parameter in your requestBody schema
+          provide the x-body-name parameter in your operation 
+          (or non-OAS compliant legacy position within the requestBody schema)
```

Comment:
I don't think that the current position is necessarily breaking OAS by design, rather, the example of using it alongside a `$ref` at the same level is